File: CS/CalDAVServer.FileSystemStorage.AspNet/Acl/PrincipalProperties.cs
using ITHit.WebDAV.Server;

namespace CalDAVServer.FileSystemStorage.AspNet.Acl
{
    /// <summary>
    /// Contains description of principal's properties.
    /// </summary>
    public static class PrincipalProperties
    {        
        public static readonly PropertyName FullName = new PropertyName("full-name", "ithit");
        public static readonly PropertyName Description = new PropertyName("description", "ithit");
        public static readonly PropertyName[] ALL;
        
        static PrincipalProperties()
        {
            ALL = new[] { FullName, Description };
        }        
    }
}

File: VB/CalDAVServer.FileSystemStorage.AspNet/Acl/PrincipalProperties.vb
Imports ITHit.WebDAV.Server

Namespace Acl

    ''' <summary>
    ''' Contains description of principal's properties.
    ''' </summary>
    Module PrincipalProperties

        Public ReadOnly FullName As PropertyName = New PropertyName("full-name", "ithit")

        Public ReadOnly Description As PropertyName = New PropertyName("description", "ithit")

        Public ReadOnly ALL As PropertyName()

        Sub New()
            ALL = {FullName, Description}
        End Sub
    End Module
End Namespace
